bundles/org.openhab.binding.coolmasternet/src/main/java/org/openhab/binding/coolmasternet/internal/CoolMasterNetBindingConstants.java

```diff
@@ -30,16 +29,16 @@
 
     public static final String BINDING_ID = "coolmasternet";
 
-    // List of all Thing Type UIDs
+    // list of all Thing Type UIDs
     public static final ThingTypeUID THING_TYPE_CONTROLLER = new ThingTypeUID(BINDING_ID, "controller");
     public static final ThingTypeUID THING_TYPE_HVAC = new ThingTypeUID(BINDING_ID, "hvac");
 
-    // List of all Channel ids
+    // list of all Channel ids
     public static final String ON = "on";
     public static final String MODE = "mode";
     public static final String SET_TEMP = "set_temp";
     public static final String FAN_SPEED = "fan_speed";
-    public static final String LOUVRE = "louvre_angle";
+    public static final String LOUVRE = "louvre";
```

Comment:
Was this a bug?

Reply:
Yes. The introduction of the `switch` statement `default` clause found it. I noted it in the commit log:

* Fix incorrect constant for louve channel